# lidar_topic: "/velodyne_points"
lidar_topic: "/lslidar_point_cloud"
ground_topic: "/cloud/ground_cloud"
no_ground_topic: "/cloud/no_ground_cloud"

is_downSample: false
voxel_leaf_size: 1.0

is_clipRadius: true
minX: -5.0
maxX: 20.0
minY: -10.0
maxY: 10.0

is_clipHeight: true
minClipHeight: 2.0

grid_size: 0.2
threshold_desc: 0.5
threshold_var: 0.05